config/initializers/paperclip_string.rb

```diff
@@ -4,6 +4,17 @@
 # Paperclip.options[:command_path] = "/usr/bin/identify"
 # Paperclip.options[:command_path] = "/usr/local/bin"
 
+Paperclip::Attachment.default_options[:storage] = :fog
+Paperclip::Attachment.default_options[:fog_directory] = ENV["GOOGLE_STORAGE_BUCKET_NAME"] || ''
+Paperclip::Attachment.default_options[:path] = ":rails_root/public/system/public/system/:class/:attachment/:id_partition/:style/:filename"
```

Comment:
Sourcing from `/app/models/spectrum.rb`, this should be:

```suggestion
Paperclip::Attachment.default_options[:path] = ":rails_root/public/system/:attachment/:id/:style/:filename"
```

Or it may be `id_partition` instead of `id`. Let's try `id` first.

Reply:
And we may need to again duplicate `public/system` to `public/system/public/system` so let's see there once this runs.